shared/actions/login/index.js

```diff
@@ -116,7 +118,7 @@ export function login (): AsyncAction {
                 payload: error
               })
 
-              if (!isRPCCancelError(error)) {
```

Comment:
and also the `desc` it was checking I think is different when the service gives it back to us, so doing the `our.desc === error.desc` wouldn't work.